.gitlab/pipeline/.gitlab-ci.yml

```diff
@@ -296,3 +297,11 @@ php-copypaste-check:
     - php-lint-php7.4
   script:
     - composer ci:php:copypaste
+
+composer-normalize:
+  extends: .default
+  stage: codestyle
+  needs:
+    - build-composer-dependencies
```

Comment:
This step does not need the Composer dependencies.

Reply:
that's true :)